polynote-kernel/src/main/scala/polynote/kernel/interpreter/python/PythonInterpreter.scala

```diff
@@ -551,8 +551,11 @@ class PythonInterpreter private[python] (
       |                # Lifted from IPython.core.ultratb
       |                def get_chained_exception(exception_value):
       |                    cause = getattr(exception_value, '__cause__', None)
+      |                    if cause == None:
+      |                        cause = getattr(exception_value, 'cause', None)
```

Comment:
Nice catch! Where does this `cause` key come from? (as opposed to `__cause__`)?

Reply:
I dunno, I guess the idea was that if it's a JVM exception you'd have `cause`.

Reply:
I'm happy to revert this; not sure it actually achieved anything. Might be left over from trying to figure out why pyspark's exceptions had changed.

Reply:
Hmm, is it possible for a JVM exception to get here somehow? py4j/spark weirdness? If so, makes sense to keep. Otherwise I guess it's one extra call which probably doesn't hurt too much 🤷

Reply:
I'm pretty sure this would never actually help. I'll revert it so it doesn't rot.

Reply:
Removed this